src/core/include/openvino/opsets/opset.hpp

```diff
@@ -10,7 +10,18 @@
 #include <set>
 #include <utility>
 
+#ifndef IN_OV_COMPONENT
+#    define IN_OV_COMPONENT
+#    define WAS_OV_LIBRARY_DEFINED_SERIALIZE
+#endif
+
 #include "ngraph/factory.hpp"
```

Comment:
As I can see, we want to get rid of ngraph factory in scope of CVS-130103
Maybe it is reasonable to unite these two PRs?